### DISCUSSION TITLE
neo-tree open directories with "o"

### DISCUSSION THREAD

**Discussion by manythrough** [LazyVim/LazyVim#1554] [Q&A] [answered]
hi there!

how can i open directories in neo-tree with the "o" key?

i have added the following to the plugins/example.lua, but it is not working:

```
return {
    {
        "nvim-neo-tree/neo-tree.nvim",
        keys = {
            --{"o", false},
            {"o", "open", remap = true}
        },
    },
}
```

**Comment by dpetka2001** [comment, marked as answer]
Try the following in a file in `~/.config/nvim/lua/plugins/` directory:
```lua
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      window = {
        mappings = {
          ["o"] = "open",
        },
      },
    },
  },
}
```
You might want to put it in another file than `example.lua`, cuz this file doesn't return anything, since it's supposed to be an example and returns just an empty table.

**Comment by baddate** [comment]
don't use `example.lua` if you are using the starter template, ~~configuring it in `config/keymaps.lua` instead.~~
btw, LazyVim included the` neo-tree`
![image](https://github.com/LazyVim/LazyVim/assets/37013819/00dc2c7f-a972-4157-8e11-bcb5fcece764)

edit: configuring like this is working for me.
![image](https://github.com/LazyVim/LazyVim/assets/37013819/9fb7153b-e3e8-461f-923c-761f0590c284)

**Reply by dpetka2001** [reply]
I don't believe it should be configured in `config/keymaps.lua`, because this is an internal mapping of the plugin itself, thus it should be configured in the spec of the plugin itself.

**Reply by baddate** [reply]
you're right, maybe you should try to config in file like `plugins/neo-tree-remap.lua`?
